.sandbox/provision/tf-lifecycle/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "public_dns" {
+  value = length(aws_instance.vm) > 0 ? aws_instance.vm[0].public_dns : null
+}
+
+output "public_ip" {
+  value = length(aws_instance.vm) > 0 ? aws_instance.vm[0].public_ip : null
+}
+
+output "password" {
+  value = length(aws_instance.vm) > 0 ? rsadecrypt(aws_instance.vm[0].password_data, file(var.keypair_file)) : null
+}
```

Comment:
Probably add `sensitive = true`, otherwise the password will be shown in UI in plaintext.